public/styles/_home.scss

```diff
@@ -37,7 +37,7 @@
 }
 
 a.featured-app-list-item {
-  background-color: #e4e4e4;
+  background-color: $main-bg-color;
```

Comment:
I think these should stay darker to provide contrast to the (mostly white) screenshots. This change reduces visibility for the screenshots.

![image](https://user-images.githubusercontent.com/16010076/95131018-45653b00-0712-11eb-975c-7282e611f5f2.png)

Reply:
Updated per this feedback plus added another color that I missed the first time.